.github/workflows/test.yml

```diff
@@ -1,6 +1,10 @@
 name: test
 
-on: workflow_dispatch
+on: [push]
+
+concurrency:
+  group: ${{github.workflow}}-${{github.ref}}
+  cancel-in-progress: true
 
 env:
   FOUNDRY_PROFILE: ci
```

Comment:
Can't we do the env stuff here rather than creating a file?

Reply:
yes, fixed